modules/UI.js

```diff
@@ -0,0 +1,54 @@
+const booksContainer = document.querySelector('.book-list-container');
+
+export default class UI {
+  // Get Books from Local Storage and Updates the UI
+  static updateBooks(booksContainer) {
+    const books = JSON.parse(localStorage.getItem('books'));
+    booksContainer.innerHTML = '';
+    if (books) books.forEach((book) => this.addBook(book));
+  }
+
+  // Creates the UI's Book
+  static addBook(book) {
+    const bookUI = document.createElement('li');
+    bookUI.className = 'book';
+    const BookId = (book.title + book.author).replace(/\s/g, '');
+    bookUI.setAttribute('id', (BookId));
+    bookUI.innerHTML = `
+                <p class="book-title">"
+                    <span id='${BookId}title'>${book.title}</span>" by 
+                    <span id='${BookId}author'>${book.author}</span>
+                </p>
+                <button class="${BookId}btn remove-button" type="button">
+                    Remove
+                </button>`;
+    booksContainer.append(bookUI);
+  }
+
+  // Removes the Book
+  static removeBook(BookId) {
+    document.getElementById(BookId).remove();
+  }
+
+  // set error msg
+  static displayError(message) {
+    const errorMessage = document.querySelector('.error-message');
+    errorMessage.innerHTML = message;
+    setTimeout(() => {
+      errorMessage.innerHTML = '';
+    }, 3000);
+  }
+
+  // Check if book has been added
+  static validate(book) {
```

Comment:
Kindly use the ES6 arrow function syntax instead of the function declaration where ever possible. This is one of the requirements for this milestone. ⬇️
![image](https://user-images.githubusercontent.com/33268927/213628825-94c7fa25-dd48-4f37-a071-24e1adbb809e.png)